integration_tests/test_builtin.py

```diff
@@ -1,20 +1,20 @@
-from ltypes import i32
-
```

Comment:
I think you need to keep these here, because both LPython and CPython requires it below for `i: i32`. CPython might ignore it by accident (I think it ignores local variables, but requires it for function signatures) and LPython might currently ignore it due to us not being strict about reporting errors, but the right solution is for LPython to always give an error message when `i32` is not imported.